www/shared/styles/global/nprogress.css

```diff
@@ -0,0 +1,27 @@
+#nprogress {
+    position: fixed;
```

Comment:
Any particular reason not to use our position mixin? Please consider this for the classes below as well.